.gitignore

```diff
@@ -44,10 +44,8 @@ config.yaml
 # Mac specific metadata
 .DS_store
 
-# Python artifacts
-**/__pycache__/**
```

Comment:
just found that I have some __pycache__ from [name_suffix.py](https://github.com/UCLH-Foundry/FlowEHR/blob/main/scripts/name_suffix.py) maybe this could remain gitignored